OpenRA.Mods.Common/UtilityCommands/Utilities.cs

```diff
@@ -32,7 +32,8 @@ public static class Utilities
 			{
 				try
 				{
-					map = new Map(modData, new Folder(Platform.EngineDir).OpenPackage(mapPath, modData.ModFiles));
+					// TO DO: Make this use IMap instead of Map
+					map = (Map)modData.MapLoader.Load(modData, new Folder(Platform.EngineDir).OpenPackage(mapPath, modData.ModFiles));
```

Comment:
Redundant cast. And for the comment, are there any plans?

Reply:
No plans as of yet but I think it is worth keeping for a future PR potentially.